MetaMask/metamask-mobile
# fix(3245): green checkbox vertical alignment on the create pw screen

## **Description**
Currently the green checkmark seems to be positioned at the bottom, it should probably be centered
Screenshots below were on iphone 15, should be tested on smaller, and bigger iphones, as well as on Android before merging.


## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/3245

Currently the green checkmark seems to be positioned at the bottom, it should probably be centered.
This was also tested on android, as well as on iOS min and max sized screens.

## **Manual testing steps**

1. create a new wallet
2. on the choose password screen choose matching passwords, the green checkmark should be centered vertically but it is not

## **Screenshots/Recordings**



| iOS Not OK       | iOS OK        |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/c7f6643f-eed8-4fc7-8f75-3f9743744782"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/ac56e515-5797-4129-922d-bb205be834c1"> |

| Android Not OK       | Android OK        |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/49443880-8fcc-415c-9ef3-abf3db6adb1b"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/701221a5-d774-46fe-86ca-82c214b94c7b"> |



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
